cfpb/consumerfinance.gov
# Prepaid credit card agreements: Update result count to 25 per page

See https://[GHE]/CFPB/el-camino/issues/365

## Changes

- Prepaid credit card agreements: Update result count to 25 per page


## How to test this PR

2. Search on http://localhost:8000/data-research/prepaid-accounts/search-agreements/ should have 25 results or less per page. Compare to 20 results or less on production https://www.consumerfinance.gov/data-research/prepaid-accounts/search-agreements/
